modules/mongodb/manifests/s3backup/backup.pp

```diff
@@ -60,61 +51,50 @@
   $private_gpg_key = undef,
   $private_gpg_key_fingerprint,
   $s3_bucket  = 'govuk-mongodb-backup-s3',
-  $server = 'localhost',
-  $standalone  = False,
   $user = 'govuk-backups'
-  ){
+){
 
   validate_re($private_gpg_key_fingerprint, '^[[:alnum:]]{40}$', 'Must supply full GPG fingerprint')
 
-# create user
+  # create user
   user { $user:
     ensure     => 'present',
     managehome => true,
   }
 
-# push env files
+  # push env files
   file { [$env_dir,"${env_dir}/env.d"]:
     ensure => directory,
     owner  => $user,
     group  => $user,
     mode   => '0770',
   }
 
-    file { "${env_dir}/env.d/AWS_SECRET_ACCESS_KEY":
-      content => $aws_secret_access_key,
-      owner   => $user,
-      group   => $user,
-      mode    => '0660',
-    }
-
-    file { "${env_dir}/env.d/AWS_ACCESS_KEY_ID":
-      content => $aws_access_key_id,
-      owner   => $user,
-      group   => $user,
-      mode    => '0640',
-    }
-
-    file { "${env_dir}/env.d/AWS_REGION":
-      content => $aws_region,
-      owner   => $user,
-      group   => $user,
-      mode    => '0640',
-    }
-
-  # push scripts
-  file { '/usr/local/bin/mongodb-backup-s3':
-    ensure  => present,
-    content => template('mongodb/mongodb-backup-s3.erb'),
+  file { "${env_dir}/env.d/AWS_SECRET_ACCESS_KEY":
+    content => $aws_secret_access_key,
     owner   => $user,
     group   => $user,
-    mode    => '0755',
-    require => User[$user],
+    mode    => '0660',
```

Comment:
I'd suggest master is wrong, doesn't make sense to have one file different to the others